README.md

```diff
@@ -12,6 +12,11 @@ on how to do that, including how to develop and test locally and the versioning
 
 ## Release Notes
 
+### version 1.18.1
```

Comment:
This is probably going to be 1.19.0, merged with the other changes now in develop, but generally I just leave this as TBD until making the release.